### DISCUSSION TITLE
Discord server is up!

### DISCUSSION THREAD

**Discussion by MarcinOrlowski** [logisim-evolution/logisim-evolution#1072] [Announcements]
We are running experimental Discord server to see how this fits in to the project. [Join now!](https://discord.gg/AQFzQJKUcZ) and let us know what you think!

![discord](https://user-images.githubusercontent.com/8041294/132833310-f98fdfc5-b950-4fa2-9971-22d55e45e77a.png)

**Comment by uwezi** [comment]
I just wanted to join, but the invite link is invalid...

**Reply by MarcinOrlowski** [reply]
@R3dst0ne could you please take a look and see if either we can have non-expiring link? If not, what's most common way of sharing discord serves with the community?

**Comment by R3dst0ne** [comment]
I created a non-expiring link.
https://discord.gg/AQFzQJKUcZ
